app/internal/page/attorney/guidance.go

```diff
@@ -14,8 +15,8 @@ type guidanceData struct {
 	Lpa    *page.Lpa
 }
 
-func Guidance(tmpl template.Template, donorStore DonorStore) page.Handler {
-	return func(appData page.AppData, w http.ResponseWriter, r *http.Request) error {
+func Guidance(tmpl template.Template, donorStore DonorStore) Handler {
+	return func(appData page.AppData, w http.ResponseWriter, r *http.Request, _ *actor.AttorneyProvidedDetails) error {
```

Comment:
I've not seen `_` used as an argument name before. Does this work in the same way as var names to show its unused?

Reply:
yep, just showing it isn't used